templates/web/peterborough/waste/_address_display_bulky_summary.html

```diff
@@ -0,0 +1,6 @@
+<dl class="waste__address">
+  <dt>Address</dt>
+  <dd class="govuk-!-margin-bottom-5">[% property.address %]</dd>
+  <dt>Are you the resident of this property or booking on behalf of the property resident?</dt>
```

Comment:
Given you can't get here if you've answered No now, I don't think this bit is needed?